meta-iotqa/lib/oeqa/runtime/multimedia/vaapi/test_vaapi_encode_decode.py

```diff
@@ -0,0 +1,198 @@
+import os
+from oeqa.oetest import oeRuntimeTest
+
+'''
+This test suit tests VAAPI Decoding and Encoding
+'''
+
+
+class VAAPITest(oeRuntimeTest):
+
+    '''
+    The video files have been created using gstreamer and videotestsrc as input
```

Comment:
No as we will need different gstreamer plugins to create then which are not there in image .